From: Alex Kalugin <akalugin@yandex-team.ru>
Date: Mon, 21 Aug 2023 16:54:42 +0000
Subject: Remove unused IsUrlElisionDisabled function.

Change-Id: Ib4caff934b3fd58b79ac056023e32fec21f37a03
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4793499
Auto-Submit: Alexander Kalugin <akalugin@yandex-team.ru>
Reviewed-by: Varun Khaneja <vakh@chromium.org>
Commit-Queue: Varun Khaneja <vakh@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1185917}
---
 .../user_interaction_observer.cc              | 21 -------------------
 1 file changed, 21 deletions(-)

diff --git a/chrome/browser/safe_browsing/user_interaction_observer.cc b/chrome/browser/safe_browsing/user_interaction_observer.cc
--- a/chrome/browser/safe_browsing/user_interaction_observer.cc
+++ b/chrome/browser/safe_browsing/user_interaction_observer.cc
@@ -34,27 +34,6 @@ namespace safe_browsing {
 
 WEB_CONTENTS_USER_DATA_KEY_IMPL(SafeBrowsingUserInteractionObserver);
 
-namespace {
-
-bool IsUrlElisionDisabled(Profile* profile,
-                          const char* suspicious_site_reporter_extension_id) {
-  if (profile &&
-      profile->GetPrefs()->GetBoolean(omnibox::kPreventUrlElisionsInOmnibox)) {
-    return true;
-  }
-#if BUILDFLAG(ENABLE_EXTENSIONS)
-  DCHECK(suspicious_site_reporter_extension_id);
-  if (profile && extensions::ExtensionRegistry::Get(profile)
-                     ->enabled_extensions()
-                     .Contains(suspicious_site_reporter_extension_id)) {
-    return true;
-  }
-#endif
-  return false;
-}
-
-}  // namespace
-
 // static
 const char* SafeBrowsingUserInteractionObserver::
     suspicious_site_reporter_extension_id_ = kPreventElisionExtensionId;
-- 

